pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -13,6 +13,7 @@
 , configureFlags ? []
 , description ? ""
 , doCheck ? !isCross && (stdenv.lib.versionOlder "7.4" ghc.version)
+, doBench ? false
```

Comment:
would `withBenchmarkDepends` be better? I'm not sure when we'd want to run the benchmarks in a nix build. My use case here is to make the benchmark dependencies available in the shell environment.